Unix/installbuilder/conf/omilogrotate.conf

```diff
@@ -0,0 +1,8 @@
+/var/opt/omi/log/omiserver.log {
+    rotate 10
+    missingok
+    notifempty
+    compress
+    size 100M
+    copytruncate
+}
```

Comment:
Is there any way to put comments in this file on what each field mean?

Reply:
Will add comments.
Doc: https://linux.die.net/man/8/logrotate